homeassistant/components/dwd_weather_warnings/const.py

```diff
@@ -6,9 +6,14 @@
 import logging
 from typing import Final
 
+from homeassistant.const import Platform
+
 LOGGER = logging.getLogger(__package__)
 
-CONF_REGION_NAME: Final = "region_name"
+DOMAIN: Final = "dwd_weather_warnings"
+
+CONF_OLD_REGION_NAME: Final = "region_name"
```

Comment:
I think you should leave this constant as `CONF_REGION_NAME`.

Reply:
Can do, thought I'd "highlight" it as the deprecated property.

Other topic but for my last 2 commits, the CI job for mariadb:10.10.3 for Python 3.10 failed. Do I have any influence regarding that? I don't remember changing code that has anything to do with that :/

Reply:
I think you can ignore those CI errors as they should not be relevant for this PR